### ISSUE TITLE
SyntaxError: invalid syntax - middleware/wsgi.py", line 43

### ISSUE THREAD

**Issue by omaishar** [MindscapeHQ/raygun4py#58]
middleware/wsgi.py", line 43:

```
def build_request(self, environ):
        request = {}

        http_host = environ.get('**HTTP_HOST']**, None)
```

Version 3.1.4

**Comment by fundead** [comment]
Good spotting - thanks for reporting this one. I will get a fix pushed today for this.

**Comment by fundead** [comment]
Apologies for the delay in getting this one merged - this has been released onto PyPI as v3.1.5.
